src/codeu/chat/ServerMain.java

```diff
@@ -89,13 +89,14 @@ public static void main(String[] args) {
   private static void runServer(Uuid id,
                                 Secret secret,
                                 ConnectionSource serverSource,
-                                ConnectionSource relaySource) {
+                                ConnectionSource relaySource, File persistentPath) {
 
     final Relay relay = relaySource == null ?
                         new NoOpRelay() :
                         new RemoteRelay(relaySource);
-
-    final Server server = new Server(id, secret, relay);
+    
+    File fp = new File(persistantPath, 'path.txt');
```

Comment:
Single quotes are only for characters, not strings, need `"path.txt"`. 

Please make sure you are running `python build.py build` before you commit. Almost all the bugs I comment on here could be caught if you just ran compilation before committing.